includes/classes/PullListTable.php

```diff
@@ -527,6 +527,7 @@ public function column_cb( $post ) {
 	public function get_bulk_actions() {
 		if ( empty( $_GET['status'] ) || 'new' === $_GET['status'] ) { // @codingStandardsIgnoreLine Nonce not required.
 			$actions = [
+				'-1'             => esc_html__( 'Bulk Action', 'distributor' ),
```

Comment:
Minor change but I think the text here should be `Bulk actions` to match what WordPress uses elsewhere